src/axom/inlet/examples/example1_expected_documentation.rst

```diff
@@ -0,0 +1,106 @@
+==================
+Input Deck Options
+==================
+--------------
+thermal_solver
+--------------
+----
+mesh
+----
+--
+u0
+--
+-----
+kappa
+-----
+------
+solver
+------
+
+Description: This is the solver sub-table in the thermal_solver table
+
+.. list-table:: Fields
+   :widths: 25 25 25 25 25
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Field Name
+     - Description
+     - Default Value
+     - Range
+     - Required
+   * - filename
+     - file for thermal solver
+     - Not specified

```

Comment:
Just leave these blank putting something there just confuses and clutters the output, something like this should do:

```
    - 
```